aszczachor/src/components/navigation/MobileMenu.js

```diff
@@ -0,0 +1,72 @@
+import styles from "components/navigation/mobileMenu.module.css";
+import { Link } from "react-router-dom";
+import { useState, useEffect, useRef } from "react";
+import ExpandButton from "./ExpandButton";
+
+const MobileMenu = ({
+  fashionStyling,
+  collections,
+  links,
+  openMenu,
+  menuBtn,
+}) => {
```

Comment:
Pododawaj tu chociaż jakieś PropTypes, tak to nie wiadomo co to przychodzi, żadne typy nie są pilnowane 😞 
PS. popraw to wszędzie, plisss